node_helper.js

```diff
@@ -53,10 +54,12 @@ module.exports = NodeHelper.create({
     if (notification === "ACTIVATE_MONITOR") {
       Log.info("MMM-MotionDetector: activating monitor.");
       this.activateMonitor();
-    }
-    if (notification === "DEACTIVATE_MONITOR") {
+    } else if (notification === "DEACTIVATE_MONITOR") {
       Log.info("MMM-MotionDetector: deactivating monitor.");
       this.deactivateMonitor();
-    }
+    } else {
+			console.log(this.name + ": Motion detected (" + payload.score +")");
```

Comment:
when would that ever be reached?

Reply:
IIRC, it would be if the debug flag is true and the motion is within the timeout limit.

Reply:
I think this will also be reached if any other notification from any other module is send. And then no score will be in there. You should only react to specific notifications and not general on all

Reply:
Fair enough. I'll take it out.